docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Welcome to Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial apps.
+
+### Minimum requirements:
+
+* Windows insider version >= 25357
+* [MPT v1.2023.517.0] (https://github.com/microsoft/win32-app-isolation/releases/tag/v0.1.1)
+* If is needed to know which capabilities are required, one will use ACP (check the MPT link) and [WPR] (https://learn.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-recorder)
```

Comment:
Same with the link, also we can give ACP the link address. The actual link address won't appear in the docs, only the text in `[]` will be displayed.